README.md

```diff
@@ -1,169 +1,98 @@
 # SpiceDB Postgres Connector - Under Construction
```

Comment:
Can we structure the README like all of the other projects?

```md
# Project
[badges]

[stability warning]

description

features:
- one
- two

contributing link

## Getting Started / Examples
```

Reply:
I was thinking we would do that once it's ready for general consumption?